migrate.py

```diff
@@ -459,6 +462,9 @@ def verify_github_issue_import_finished(status_url, auth, headers):
     """
     while True:  # keep checking until status is something other than 'pending'
         respo = requests.get(status_url, auth=auth, headers=headers)
+        if respo.status_code == 404:
+            print("404 retrieving status URL", status_url)
```

Comment:
Also expand this message slightly so the user understands things might not have completely blown up. Again, something like "GitHub sometimes inexplicably returns a 404 for the check url for a single issue. Sometimes these issues still import successfully, sometimes not. For details, see #77."